.bazelrc

```diff
@@ -2,7 +2,7 @@
 # flag: --noremote_accept_cached
 build:darwin --remote_http_cache=https://bazel-cache.da-ext.net
 build:linux --remote_http_cache=https://bazel-cache.da-ext.net
-build:windows-ci --remote_http_cache=https://bazel-cache.da-ext.net
+build:windows-ci --remote_http_cache=https://bazel-cache.da-ext.net/v1
```

Comment:
There's a corresponding line in `ci/configure-bazel.sh`. I think that needs to be updated too.

Reply:
Thanks, updated.